aggie-coding-club/Rev-Registration
# Reverse schedule preview course direction

## Description

Reverses the schedule preview course direction to map the reversed CourseSelectColumn order. I _could_ probably add tests to this, but it's so dang simple that I really didn't think it was necessary.

## How to test

Add two courses, generate a schedule, and ensure they match up with the CourseSelectColumn direction

## Screenshots

|Before|After|
|--|--|
|![chrome_vL6NRpSMSS](https://user-images.githubusercontent.com/7295783/107886219-5c334f80-6eb3-11eb-8678-7b30c04515d5.png)|![chrome_bQYmDWQs7E](https://user-images.githubusercontent.com/7295783/107886225-5fc6d680-6eb3-11eb-9464-205276a24994.png)|

## Related tasks

Closes #499
